_episodes/01-intro.md

```diff
@@ -22,8 +22,9 @@ keypoints:
 *   Much more to a research software project than just code:
     *   Data
     *   Organization
-    *   Communication
     *   Process
+    *   Communication
+        *   We should think about programming not as instructing a computer what to do, but as communicating with other humans what we want the computer to do (paraphrased from Donald Knuth)
```

Comment:
I actually had an entire slide dedicated to this last point, and tried to make it one of the central themes for the day. Emphasized getting away from the stereotype of socially awkward über-nerd coders to people that can actually communicate with others clearly and effectively about what computing needs to be done and why.